cmd/autofix_test.go

```diff
@@ -1,13 +1,50 @@
 package cmd
 
 import (
+	"bufio"
+	"fmt"
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-
-	"github.com/go-test/deep"
 )
 
+func compareFiles(file1, file2 string) bool {
```

Comment:
Is it worth being explicit that this only safely compares text files and will ignore EOL characters? (`\r?\n`)

Reply:
Why does this only safely compare text files? And why do you say it ignores EOL characters?

Reply:
`NewScanner` defaults to `ScanLines`. I'm still pretty new to this go jazz but `ScanLines` tries to drop carriage returns and newlines. The source seems to agree with my remembering: https://golang.org/src/bufio/scan.go?s=11799:11877#L345. 

If a non-text file is compared it would drop a bunch of probably meaningful, if misunderstood, bytes. It's a pretty unlikely situation where you end up with a false positive there in non-trivial cases I suppose.